src/data/finsupp/basic.lean

```diff
@@ -406,6 +406,10 @@ variables {b}
 
 lemma support_update_ne_zero (h : b ≠ 0) : support (f.update a b) = insert a f.support := if_neg h
 
+lemma superset_of_support_update_ne_zero {f : α →₀ M} {q : α} {b : M} (h : b ≠ 0) :
+  f.support ⊆ (f.update q b).support :=
+by simpa only [support_update_ne_zero _ _ h, subset_iff, mem_insert] using λ a ha, or.inr ha
+
```

Comment:
I'm also not convinced we need this lemma, since it's just
```
by simp only [support_update_ne_zero _ _ h, subset_insert]
```

Reply:
This one, admittedly, I just put in to go along with the other two `subset_of_support_*` lemmas.  If you think it's not useful — e.g. if there's an equally quick and obvious way to get from `x ∈ f.support` to `x ∈ (f.update q b).support` — then I'm happy to drop it.